cypress/minecraft-server.mjs

```diff
@@ -0,0 +1,26 @@
+//@ts-check
+import mcServer from 'flying-squid'
```

Comment:
Why .mjs ?

Reply:
Does it really matter?

Reply:
> Why .mjs ?

Because I like import syntax highlight.
I generally more like esm because of top level await.
And yes I'm aware of esm restrictions (__surname and so on

Reply:
We don't use .mjs anywhere else in PrismarineJS. Syntax highlighting works there though

Reply:
Oh ok if you want me to remove it won't be a problem. You just asked why…
And no, syntax highlighting doesn't work in cjs files in the same way as it works in esm (I mean import keyword!)